jsnip/src/main/java/dev/marcelo/java/Caller.java

```diff
@@ -19,15 +19,15 @@ public int getSomeInt() {
   }
 
   public void setSomeInt(int someInt) {
-    this.someInt = Callee.someInt + someInt;
+    this.someInt = Callee.someInt + someInt + 1;
   }
 
   public void testAPI() {
     int k = getSomeInt();
-    DummyAPI api = new DummyAPI(k);
+    DummyAPI api = new DummyAPI(k+1);
     RandomAPI rapi = new RandomAPI();
     String s = rapi.toString();
-    setSomeInt(api.getNum());
+    setSomeInt(api.getNum()+1);
```

Comment:
[reviewpad-sig]: <>
[Link to Comment](http://localhost:9000/review/github.com/marcelosousa/java-snippets/pull/3#comment-69d9467e-b888-4853-a1a0-b9e18586b5e6) [Link to Comment](http://localhost:9000/review/github.com/marcelosousa/java-snippets/pull/3#comment-6d32e969-4b60-45c2-b016-8eee012772ca)